README.md

```
@@ -147,18 +147,38 @@ Experiment:
-streamlit run frontend/streamlit_app.py
-```
-
----
+  # Запуск проекта
+  ## Windows
+  ### Установка pixi  ``` iwr -useb https://pixi.sh/install.ps1 | iex```
+  ### Сборка контейнеров
+  ```
+  pixi run build
+  ```
+  ### Запуск всей системы
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)